Equipment.md

```diff
@@ -124,26 +126,30 @@ __Vulnerability__ is an optional rule for players who want excessive realism. It
 
 |Armor|AC Bonus|Move Cost|Vulnerability|
 |---|---|---|---|
-|  Light Leather   |+2 |0 ft. |
+|  Light Leather   |+2 |0 ft. |5 [Slashing]
 | Light Piecemeal  |+3 |0 ft. |
 |   Light Metal    |+3 |5 ft. |
 |   Light Hybrid   |+4 |0 ft. |
-|  Medium Leather  |+5 |0 ft. |
+|  Medium Leather  |+5 |0 ft. |2 [Slashing]
 | Medium Piecemeal |+6 |0 ft. |2 [Piercing]
-|   Medium Metal   |+7 |5 ft. |
+|   Medium Metal   |+7 |5 ft. |2 [Bludgeoning]
 |  Medium Hybrid   |+8 |5 ft. |
 |  Heavy Leather   |+9 |0 ft. |
 | Heavy Piecemeal  |+10|5 ft. |5 [Piercing]
 |   Heavy Metal    |+11|10 ft.|5 [Bludgeoning]
 |   Heavy Hybrid   |+12|10 ft.|
+
+|Shield|AC Bonus|Move Cost|Vulnerability|
+|---|---|---|---|
 |Light Wood Shield |+1 |0 ft. |
-|Light Metal Shield|+1 |0 ft. |
+|Light Metal Shield|+2 |0 ft. |
 |Heavy Wood Shield |+3 |0 ft. |
-|Heavy Metal Shield|+3 |0 ft. |
-|Tower Wood Shield |+7 |5 ft. |
+|Heavy Metal Shield|+4 |5 ft. |
+|Tower Wood Shield |+7 |10 ft.|
+|Tower Metal Shield|+9 |15 ft.|
```

Comment:
1. By "Tower shield" I mean the superior kite shield, not the roman scutum. And [kite shields were made of leather or wood primarily](https://en.wikipedia.org/wiki/Kite_shield#Construction), because they were already so huge there was no reason to make them of (primarily) metal. So I disagree with Tower Metal Shield existing.
1. No shield provides as good AC bonus as full body armor of leather gambison under layered cloth. Honestly, I don't think they should get above +5 in retrospect (so bad on me, too).

Reply:
Fair enough, I'll remove the metal tower shield and drop the wood one to +5.